src/runtime/lpython_builtin.py

```diff
@@ -994,12 +1125,50 @@ def _lpython_str_isascii(s: str) -> bool:
             return False
     return True
 
-def _lpython_str_isspace(s:str) -> bool:
+
+def _lpython_str_isspace(s: str) -> bool:
     if len(s) == 0:
         return False
-    ch: str 
+    ch: str
+    ch_ord: i32
     for ch in s:
-        if ch != ' ' and ch != '\t' and ch != '\n' and ch != '\r' and ch != '\f' and ch != '\v':
+        ch_ord = ord(ch)
+        # A Unicode character is considered a 'whitespace' if it has has a bidirectional
+        # type 'WS', 'B' or 'S'; or the category 'Zs'.
+        # Check every character in the string to see if it is a whitespace.
+        # If none of the characters match, we have atleast one character which is not
+        # a whitespace, so we return `False`.
+        if (
+            ch_ord != 9
```

Comment:
Thanks for handling all the cases, but it doesn't seems to be more readable.
By readable what I meant is, I have to manually check what `9` is, and confirm that `\t` is handled.
 
Was there any issue with previous changes?
Also, instead of this long condition check, why not keep the strings in list and have a single condition check?